README.md

```diff
@@ -18,8 +21,11 @@
 
 To get a local copy up and running follow these simple example steps.
 
-1. Copy this link `https://github.com/hiromataba/Leaderboard` and type git clone `https://github.com/hiromataba/Leaderboard` in your terminal to get and run this project locally
-2. Run the live server or simply open index.html in dist folder with your browser.
+1. Copy this link `https://github.com/hiromataba/Leaderboard`
+2. In your terminal run:
+- `git clone https://github.com/hiromataba/Leaderboard` 
+3. Run `npm install` 
```

Comment:
It would be nice to include 
```
Run `npm start`
```
as step 4.

I trust you can update this before merging.